README.md

```diff
@@ -121,6 +121,7 @@ Create a Python 3.7 virtual environment, and install AllenNLP in `editable` mode
 
   ```bash
   pip install --editable .
+  pip install -r dev-requirements.txt
```

Comment:
I'd note that this is only if you want to develop on the library. Might not be obvious to a casual user.

Reply:
I think that it's best that, if users are cloning the master version of our GitHub repository locally, they simply install the developer requirements.  Most users will `pip install allennlp` (which comes earlier in the README) and will avoid the `dev-requirements.txt` (because it's not mentioned in that section at all).

I think that most people who install from source will end up needing `dev-requirements.txt` and for those who don't they shouldn't get in the way.